pages/blog/may-2019-update.md

```diff
@@ -0,0 +1,42 @@
+---
+description: Here are the most significant changes in May.
+---
+
+# May 2019 Update
+
+**Olivier Tassinari**
+
+*June 8, 2019*
+
+Here are the most significant changes in May:
+
+- 🎉 We have released Material-UI v4 stable during the React Europe conference.
+You can find the details in our [blog post 📝](https://medium.com/material-ui/material-ui-v4-is-out-4b7587d1e701) and a quick introduction in this [5 minutes talk](https://www.youtube.com/watch?v=m54_CPfbWow).
```

Comment:
Should the quick introduction come first, perhaps?

Reply:
There is a font size issue with the prez, but why not!